examples/coze-js-web/src/use-coze-api.ts

```
@@ -27,11 +30,11 @@ const useCozeAPI = () => {
   });
 
   useEffect(() => {
-    const config = JSON.parse(
+    const configData = JSON.parse(
       sessionStorage.getItem('settingConfig') || '{}',
     ) as SettingConfig;
 
-    if (config && config.authType) {
+    if (configData && configData.authType) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)